js/ai/src/model.ts

```diff
@@ -489,10 +518,15 @@ export function defineModel<
       },
       use: middleware,
     },
-    (input) => {
+    (input, ctx) => {
       const startTimeMs = performance.now();
-
-      return runner(input, getStreamingCallback(registry)).then((response) => {
+      const secondParam =
```

Comment:
What actually is the secondParam? I looks like it is context | callback | boolean | undefined... which seems really weird... am I missing something?